app/views/profiles/_form.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= form_for(@profile) do |f| %>
+  <% if @profile.errors.any? %>
+    <div id="error_explanation">
+      <h2><%= pluralize(@profile.errors.count, "error") %> prohibited this organisation from being saved:</h2>
+
+      <ul>
+      <% @profile.errors.full_messages.each do |message| %>
+        <li><%= message %></li>
+      <% end %>
+      </ul>
+    </div>
+  <% end %>
```

Comment:
We already have a partial for this, and whats more, we are even rendering it! These error things can just be deleted.